cpp/ql/src/semmle/code/cpp/commons/File.qll

```diff
@@ -23,16 +23,16 @@ predicate fopenCall(FunctionCall fc) {
  */
 predicate fcloseCall(FunctionCall fc, Expr closed) {
   exists(Function f | f = fc.getTarget() |
-    f.hasGlobalName("fclose") and
+    f.hasGlobalOrStdName("fclose") and
     closed = fc.getArgument(0)
     or
-    f.hasGlobalName("close") and
+    f.hasGlobalOrStdName("close") and
```

Comment:
There's no `std::close` either, and I presume there's also no `std::_close` (below).